### PR TITLE
Add `labels` property to `AxesActor` for convenience

### PR DESCRIPTION
**Pull request by user27182**
### Overview

Follow-up from #5029.

New property to make it more convenient to set the labels, e.g. 
``` python
axes.labels = 'UVW'
```
instead of
``` python
axes.x_axis_label = 'U'
axes.y_axis_label = 'V'
axes.z_axis_label = 'W'
```

### CODE REVIEW

File: pyvista/plotting/axes_actor.py [hunk 1]
```diff
@@ -402,6 +403,43 @@ def tip_type(self, tip_type: Union[TipType, int]):  # numpydoc ignore=GL08
...
+
+        Examples
+        --------
+        Set single-character axes labels.
+
+        >>> import pyvista as pv
+        >>> axes_actor = pv.AxesActor()
+        >>> axes_actor.labels = 'UVW'
+        >>> axes_actor.labels
+        ('U', 'V', 'W')
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:425]: The latter expression feels convenient, but this one feels confusing when viewed as a user. A short-sighted user like myself might specify “X AxisY AxisZ Axis” and so on. How about remove this and use only latter expression?

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Thanks for this.

File: pyvista/plotting/axes_actor.py (hunk 1 above)
Comment (user27182) [reply pyvista/plotting/axes_actor.py:425]: Do you mean allow `Sequence[str]` but do not allow `str`? I do agree it is perhaps a bit too convenient. We can remove it.

Comment (tkoyama010) [reply pyvista/plotting/axes_actor.py:425]: Yes. Let's remove it.

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Pull request merged**